### ISSUE TITLE
Show values of number cells in addition to bars

### ISSUE THREAD

**Issue by ginihumer** [lineupjs/lineupjs#407]
Sometimes it might be usefull to show values of number cells by default (like they are shown when selecting a row).
Currently I only found the "numberWithValues" renderer option, which shows numbers, but hides bars.

current behaviour:

- show bars; show bars + values when highlighted:
![image](https://user-images.githubusercontent.com/45741696/108343334-1d1b3c00-71dc-11eb-85b8-875a50608b72.png)

- show values, but no bars when using "numberWithValues" renderer
![image](https://user-images.githubusercontent.com/45741696/108343258-070d7b80-71dc-11eb-8d22-356a1897d7ac.png)

**Comment by puehringer** [comment]
@ginihumer Feel free to register a custom renderer which instantiates the renderer using the `renderValue: true`. 
 https://github.com/lineupjs/lineupjs/blob/master/src/renderer/BarCellRenderer.ts#L18

Would this solve the issue?

**Comment by ginihumer** [comment]
@puehringer yes, I guess this would solve the problem. I just thought that this might be a relevant feature for lineup in general.

**Comment by sgratzl** [comment]
see also #491
